builtin/common/settings/dlg_change_mapgen_flags.lua

```diff
@@ -95,20 +116,30 @@ local function get_formspec(dialogdata)
 			for noise settings in the settings menu. ]]
 			.. fgettext("defaults") .. ";" -- defaults
 			.. tostring(flags["defaults"] == true) .. "]" -- to get false if nil
+			.. "tooltip[cb_defaults;" .. fgettext("This flag overrides the eased flag based on the noise map type.")
+			.. "\n" .. fgettext("2D noise will make eased on.")
+			.. "\n" .. fgettext("3D noise will make eased off.") .. "]"
```

Comment:
Is `make` a correct expression? I am not a native speaker, but do think that something along these lines:

```suggestion
			.. "tooltip[cb_defaults;" .. fgettext("This flag overrides the 'eased' flag based on the noise type.")
			.. "\n" .. fgettext("2D noise: flag enabled.")
			.. "\n" .. fgettext("3D noise: flag disabled.") .. "]"
```

Reply:
changed it to "will force eased to be..."

Reply:
fixed